packages/kbn-test/src/jest/run.ts

```diff
@@ -145,6 +145,11 @@ export function runJest(configName = 'jest.config.js') {
     process.env.NODE_ENV = 'test';
   }
 
+  if (!process.env.REACT_VERSION) {
+    // Default to React 18 for Jest tests
+    process.env.REACT_VERSION = '18';
```

Comment:
Required because eui... exposes export compatible with react 18, when the env variable `REACT_VERSION` is set to `18`, for example; https://github.com/elastic/eui/blob/main/packages/eui/src/test/rtl/render_hook.ts#L18-L24